groups/sig-release/groups.yaml

```diff
@@ -412,26 +412,28 @@ groups:
       - guineveresaenger@gmail.com # 1.22 Emeritus Advisor
       - tpepper@vmware.com # Release Team subproject owner
     members:
-      - alessandro.vozza@microsoft.com # 1.21 Bug Triage Shadow
-      - arunmk@gmail.com # 1.21 RT Enhancements Shadow
       - ashnehete@gmail.com # 1.22 Release Docs Shadow
       - carlisia@grokkingtech.io # 1.22 Release Docs Shadow
-      - desponda@vmware.com # 1.21 Bug Triage Shadow
-      - evelyn.cupil.garcia@duke.edu # 1.21 Comms Shadow
-      - james@jameslaverack.com # 1.21 RT Enhancements Shadow
-      - joseph.r.sandoval@gmail.com # 1.21 RT Enhancements Shadow
-      - justinleegarrison@gmail.com # 1.21 Comms Shadow
-      - kcmartin2@mac.com # 1.21 Bug Triage Shadow
-      - kefroden@gmail.com # 1.21 RT Enhancements Shadow
-      - max@koerbaecher.io # 1.21 CI Signal Shadow
-      - meloncattiel@gmail.com # 1.21 Release Notes Shadow
-      - monmonmsc@gmail.com # 1.21 Bug Triage Shadow
-      - peeyushgupta91@gmail.com # 1.21 Comms Shadow
-      - pmmalinov01@gmail.com # 1.21 Release Notes Shadow
-      - priyankahariharan421@gmail.com # 1.21 CI Signal Shadow
       - rpanjwani@vmware.com # 1.22 Release Docs Shadow
-      - saumya00@outlook.com # 1.21 CI Signal Shadow
-      - snichols@vmware.com # 1.21 CI Signal Shadow
-      - soniasingla.1812@gmail.com # 1.21 Release Notes Shadow
       - striker57@gmail.com # 1.22 Release Docs Shadow
-      - xander@grzy.dev # 1.21 Comms Shadow
+      - cicih@google.com # 1.22 Release Notes Shadow
+      - Damanarora@cmail.carleton.ca # 1.22 Release Notes Shadow
+      - sladynnunes98@gmail.com # 1.22 Release Notes Shadow
+      - simran.thind@outlook.com # 1.22 Release Notes Shadow
+      - salahi.hossein@gmail.com # 1.22 CI Signal Shadow 
+      - soniasingla.1812@gmail.com # 1.22 CI Signal Shadow
+      - ania0102@gmail.com # 1.22 CI Signal Shadow
+      - ramses.green.2@gmail.com # 1.22 CI Signal Shadow
+      - voigt.christoph@gmail.com # 1.22 Bug Triage Shadow
+      - gossanth@gmail.com # 1.22 Bug Triage Shadow
+      - taylorchprr@gmail.com # 1.22 Bug Triage Shadow
+      - jgavinray@linux.com # 1.22 Bug Triage Shadow
+      - kunalkushwaha453@gmail.com # 1.22 Comms Shadow
+      - rajula96reddy@gmail.com # 1.22 Comms Shadow
+      - butlerjl@amazon.com # 1.22 Comms Shadow
+      - klkfr@amazon.com # 1.22 Comms Shadow
+      - joseph.r.sandoval@gmail.com # 1.22 Enhancements Shadow
+      - xander@grzy.dev # 1.22 Enhancements Shadow
+      - supriyapremkumar1@gmail.com # 1.22 Enhancements Shadow
+      - nng.grace@gmail.com # 1.22 Enhancements Shadow
+
```

Comment:
Thanks @guineveresaenger :)
Enhancements shadow list is missing Rey Lejano. Can you add his email as well?

Reply:
Ping @guineveresaenger